kod/object/active/holder/nomoveon/battler/monster.kod

```diff
@@ -825,7 +960,7 @@ messages:
          AND isClass(poTarget,&User)
       {
          Send(poTarget,@SubtractFromMonsterChasers,
-              #level=viLevel);
+              #level=Send(self,@GetLevel));
```

Comment:
Use the variable here, not a message call to self. Additionally, we should not be modifying this variable in any way after monster creation - this particular message call to Player is used to keep track of how many monsters are chasing the player. The viLevel (in this pull, piLevel) variable is used and must stay constant.